### PR TITLE
Add jQuery Once

### PR DESCRIPTION
**Pull request by RobLoach**
[jQuery Once](http://github.com/robloach/jquery-once) allows acting on jQuery elements only once.

### CODE REVIEW

File: files/jquery-once/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Rob Loach"
+github = "https://github.com/RobLoach/jquery-once"
+homepage = "https://github.com/RobLoach/jquery-once"
+description = "Act on jQuery elements only once."
+mainfile = "jquery.once.js"
```
Comment (jimaek) [inline files/jquery-once/info.ini]: Use the min file here `jquery.once.min.js`

File: files/jquery-once/update.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "packageManager": "github",
+  "name": "jquery-once",
+  "repo": "RobLoach/jquery-once",
+  "files": {
+    "include": [
+      "jquery.once.js",
```
Comment (jimaek) [inline files/jquery-once/update.json]: No need to include the unminified file

**Comment by jimaek** [comment]
Please rename the directory to `jquery.once`

**Comment by RobLoach** [comment]
> @jimaek  Please rename the directory to jquery.once

Renamed... Does this conflict with module loading? It is named `jquery-once` everywhere else.

**Comment by jimaek** [comment]
Its part of our file-structure rule https://github.com/jsdelivr/jsdelivr/blob/master/CONTRIBUTING.md#file-structure
>If the project is a plugin of a library, prepend the name of the library, like jquery.blurjs or bootstrap.select.

Thank you

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by RobLoach** [comment]
> @jimaek jsdelivrbot squash and merge

Damn, that was so cool. :sunglasses:

**Comment by jimaek** [comment]
We have lots of cool stuff https://hacks.mozilla.org/2014/03/jsdelivr-the-advanced-open-source-public-cdn/ :)

**Pull request closed without merging**